dg/commands/ibmcloud/generate_api_key.py

```diff
@@ -19,12 +19,12 @@
 import dg.config
 import dg.lib.ibmcloud.iam
 
+from dg.config import data_gate_configuration_manager
```

Comment:
You already "import dg.config" 2 lines before so I believe here you should import the function you specifically want (it's the only one used apparently) : 

```suggestion
from dg.config.data_gate_configuration_manager import get_ibmcloud_cli_path
```

Reply:
I removed `import dg.config` and replaced `dg.config.data_gate_configuration_manager` with `data_gate_configuration_manager`.